### PR TITLE
Fix quaternion initialisation segfault under -march=native

### LINKED ISSUE #231: Quaternion leads to segfault with -march=native
I have compiled eigenpy with clang-10 (Release mode) and the following flags: `-march=native -mfpmath=sse`. Then, the following code leads to a segfault:
```python
import eigenpy
eigenpy.Quaternion.Identity()
```

### PR DESCRIPTION
**Pull request by wxmerkt**
Relates to and resolves #231

### CODE REVIEW

File: include/eigenpy/quaternion.hpp [hunk 1]
```diff
@@ -263,24 +268,42 @@ namespace eigenpy
...
-    
-    static Quaternion* FromTwoVectors(const Vector3& u, const Vector3& v)
+
+    static Quaternion* FromCoefficients(Scalar w, Scalar x, Scalar y, Scalar z)
+    {
+      Quaternion* q(new Quaternion(w, x, y, z));
+      return q;
+    }
+
+    static Quaternion* FromAngleAxis(const AngleAxis& aa)
```
Comment (wxmerkt) [inline include/eigenpy/quaternion.hpp:278]: Note, this won't compile with `Eigen::Ref<AngleAxis>`

Comment (jcarpent) [reply include/eigenpy/quaternion.hpp:278]: Why do you want to rely on `Eigen::Ref<AngleAxis>`?

**Review by jcarpent — APPROVED** [review]
Seems good to me.

**Review by jcarpent — COMMENTED** [review]
The copy constructor works well?

File: include/eigenpy/quaternion.hpp (hunk 1 above)
Comment (wxmerkt) [reply include/eigenpy/quaternion.hpp:278]: Right, it doesn't matter here. `const Eigen::Ref<const TYPE>&` is my first approach out of habit

**Comment by cmastalli** [comment]
I confirm that it fixed the issue from my side.
Thanks @wxmerkt ;)

**Pull request merged**